compilers/concrete-compiler/compiler/lib/Bindings/Python/concrete/compiler/compilation_feedback.py

```diff
@@ -135,9 +135,12 @@ def count_per_tag(self, *, operations: Set[PrimitiveOperation]) -> Dict[str, int
             if statistic.operation not in operations:
                 continue
 
-            tag, _, _ = re.match(REGEX_LOCATION, statistic.location).groups()
-            # remove the @
-            tag = tag[1:] if tag else ""
+            try:
+                tag, _, _ = REGEX_LOCATION.match(statistic.location).groups()
+                # remove the @
+                tag = tag[1:] if tag else ""
+            except Exception:  # pylint: disable=broad-except
```

Comment:
More precise exception pattern. Broad except should not be used and try block should be minimized. E.g.

Reply:
I've removed the try except block completely